db/migrate/20150328194959_create_reviews.rb

```diff
@@ -3,8 +3,8 @@ def change
     create_table :reviews do |t|
       t.integer  :rating, null: false
       t.text     :body, null: false
+      t.integer  :sitter_id
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.